### PR TITLE
Performance Profiler: update history placeholder

### PR DESCRIPTION
**Pull request by gcsecsey**
Closes https://github.com/Automattic/dotcom-forge/issues/8998

## Proposed Changes

* Update the history chart placeholder

## Why are these changes being made?

https://github.com/Automattic/dotcom-forge/issues/8998

## Testing Instructions

* Start a new test for a site without CrUX data, or use this report URL:
```
/speed-test-tool?url=https%3A%2F%2Fgcsecseyfree.wpcomstaging.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzQ3NH0.3NqZhZmRj6_aW3TexJ_RO3CU8wiqoK2mz69314mvDrQ
```
* Check the history chart styling and the updated copy

| Before | After |
|--------|--------|
| image | CleanShot 2024-09-11 at 11 43 26@2x |

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Comment by epeicher** [comment]
HI @gcsecsey, I have added a commit [e533c35](https://github.com/Automattic/wp-calypso/pull/94373/commits/e533c35d5a4e8d0512ed100d3ccc0d8757fc9c82) to use the `translate` function for the message and update some styles. Please see a video here

https://github.com/user-attachments/assets/9a3d4f01-aef8-43ea-a1f7-123f497efd5e

The `text-wrap` works on most browsers, but the default for older browsers is `wrap` which is a good default. Please let me know if you agree with that

**Review by gavande1 — APPROVED** [review]
Thanks @gcsecsey and @epeicher for working on this. Thanks!

File: client/performance-profiler/components/charts/history-chart.jsx [hunk 1]
```diff
@@ -236,13 +226,7 @@ const HistoryChart = ( { data, range, height } ) => {
...
 				{ ! dataAvailable && (
 					<div className="info">
-						History unavailable
-						<Icon
-							onMouseOver={ handleInfoToolTip }
-							onMouseOut={ handleHideToolTip }
-							icon={ info }
-							className="icon"
-						/>
+						{ translate( 'Historical data will appear after more test runs' ) }
```
Comment (gavande1) [inline client/performance-profiler/components/charts/history-chart.jsx]: I just remembered that this is not true. It won't appear automatically unless we make changes on the backend. I believe we haven't planned that yet.

Comment (gcsecsey) [reply client/performance-profiler/components/charts/history-chart.jsx]: So just to clarify, we will only be able show the history when the site has CrUX data, right?

Comment (gavande1) [reply client/performance-profiler/components/charts/history-chart.jsx]: Exactly. It will only be available when CruX data is available.

Comment (gcsecsey) [reply client/performance-profiler/components/charts/history-chart.jsx]: Okay, in that case, until we iterate on this, I think we should update the copy to suggest that there's no historical data available for the specific site. @matt-west wdyt would be the most appropriate copy here?

Comment (gcsecsey) [reply client/performance-profiler/components/charts/history-chart.jsx]: Sorry, I just saw your message here: p1726048080075789/1725969661.316119-slack-C07AV9N4Y1H

**Comment by matt-west** [comment]
Let's be clearer in the empty state about exactly why data is not available.

> No history available
> The Chrome User Experience Report collects speed data from real site visits. Low-traffic sites may not generate enough data to provide historical trends.

I’ve updated the designs in Figma.

image

**Comment by gcsecsey** [comment]
I made the changes and updated the screenshot in the PR description for translation.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16701076).

**Pull request merged**